.travis.yml

```diff
@@ -17,25 +17,25 @@ jobs:
     - stage: Test
```

Comment:
`.travis.yml` should probably be removed completly?

Reply:
Or were you thinking that we'd keep travis around a little longer to make use the Actions are working correctly.

Reply:
> .travis.yml should probably be removed completly?

As you prefer. We can remove it or keeping aside gh actions for like a couple of weeks just to spot potential degradations